File: FiberDeskApp/app/src/main/java/com/example/fiberdesk_app/Ticket.kt
package com.example.fiberdesk_app

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Ticket(
    val folio: String,
    val cliente: String,
    val prioridad: String,
    val asunto: String,
    val tecnico: String,
    val creadoPor: String,
    val estado: String,
    val fecha: String,
    val descripcion: String,
    val archivado: Boolean = false
) : Parcelable

File: FiberDeskAPI/models/Ticket.js
const mongoose = require("mongoose");

const TicketSchema = new mongoose.Schema({
    folio: { type: String, required: true, unique: true },
    cliente: String,
    prioridad: String,
    asunto: String,
    tecnico: String,
    creadoPor: String,
    estado: { type: String, default: "Pendiente" },
    fecha: String,
    descripcion: String,
    archivado: { type: Boolean, default: false } 
});

module.exports = mongoose.model("Ticket", TicketSchema);
 